frontend/packages/console-app/src/components/volume-snapshot/create-volume-snapshot/create-volume-snapshot.tsx

```diff
@@ -170,11 +159,33 @@ const CreateSnapshotForm = withHandlePromise<SnapshotResourceProps>((props) => {
   }, [namespace, pvcName]);
 
   const [data, loaded, loadError] = useK8sWatchResource<PersistentVolumeClaimKind[]>(resourceWatch);
+  const scList = scObjListLoaded ? scObjList.items : [];
+  const snapshotClassFilter = React.useCallback(
+    (snapshotClass: VolumeSnapshotClassKind) => scObj?.provisioner.includes(snapshotClass?.driver),
+    [scObj],
+  );
+  const filterWrapper = (storageClassObj) => (snapshotClass: VolumeSnapshotClassKind) =>
```

Comment:
this is the function as `snapshotClassFilter`. You should use just `snapshotClassFilter` in `useEffect` below

Reply:
added to useEffect